cmd/gaiad/cmd/root.go

```diff
@@ -149,6 +157,9 @@ func txCommand() *cobra.Command {
 		authcmd.GetBroadcastCommand(),
 		authcmd.GetEncodeCommand(),
 		authcmd.GetDecodeCommand(),
+		vestingcli.GetTxCmd(),
+		feegrantcli.GetTxCmd(),
+		authzcli.GetTxCmd(),
```

Comment:
ditto. I think we don't need to call these.

Reply:
let's remove these too @yaruwangway